From c42c6a8363f05cb7a19d28302c1fda2f7b305b4e Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 8 Sep 2020 21:00:28 +0800
Subject: [PATCH] Test case for Delete included .h changing - nasm file

Description: Delete included ForHAddTest.h file in the MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm b/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
index 38ab5e654a..4f9b98f18b 100644
--- a/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
+++ b/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
@@ -6,7 +6,7 @@
 ;  SPDX-License-Identifier: BSD-2-Clause-Patent
 ;
 ;**/
-#include "ForHAddTest.h"
+
     SECTION .text
 
 ;
@@ -30,7 +30,7 @@ global ASM_PFX(AsmVectorFixup)
 
 global ASM_PFX(AsmGetVectorTemplatInfo)
 ASM_PFX(AsmGetVectorTemplatInfo):
-        mov   ecx, [esp + testaddr]
+        mov   ecx, [esp + 4]
         mov   dword [ecx], @VectorTemplateBase
         mov   eax, (@VectorTemplateEnd - @VectorTemplateBase)
         ret
-- 
2.27.0.windows.1

